fmriprep/utils/bids.py

```diff
@@ -195,8 +195,8 @@ def validate_input_dir(exec_env, bids_dir, participant_label):
             "MALFORMED_BVAL",
             "MISSING_TSV_COLUMN_EEG_ELECTRODES",
             "MISSING_SESSION",
+            "NO_T1W",
         ],
-        "error": ["NO_T1W"],
```

Comment:
This is pretty useful to have, in general. We should think about if there's a way to tell when we get a T1w image from the anatomical derivatives, and only disable it in that case.

Reply:
I tried doing that at first, but this is called at argument parsing stage https://github.com/nipreps/fmriprep/blob/d85fb53007ec2b8f00624ec8d32658ca1bc3c78c/fmriprep/cli/parser.py#L828 so we have no clue about what the derivatives contain.

Reply:
The simple thing we can do now is if any `--derivatives` is passed we make it a warning, otherwise an error.
That way it will be enforced in most standard usecases.

The alternative would be to relocate such check at workflow build stage, but that seems to defeat the purpose of checking early for input compliance.

Reply:
Yeah, that makes sense to me.